spec/spec_helper.rb

```diff
@@ -1,8 +1,31 @@
-require 'rspec-puppet'
+require 'puppetlabs_spec_helper/module_spec_helper'
+require 'rspec-puppet-facts'
+include RspecPuppetFacts
+add_custom_fact :pe_version, '2016.4.2'
```

Comment:
I think this needs to be added in .sync.yml as a `spec_override` as `default_module_facts.yml` is a flat file that modulesync will override every time.